doc/service-tests.md

```diff
@@ -7,255 +7,249 @@ automated tests should be included. They serve three purposes:
+3. `t` is exported from the module.
 
-The JSON format for this badge is `{ name: 'build', value: 'passing' }`.
+### (2) Our First Test Case
 
-Here's what our first test looks like:
+First we'll add a test for the typical case:
 
 ```js
-t.create('build status on default branch')
-  .get('/rust-lang/rust.json')
-  .expectJSONTypes(Joi.object().keys({
-    name: 'build',
-    value: Joi.equal('failing', 'passing', 'unknown')
-  }));
+const { isBuildStatus } = require('../test-validators')
+
+t.create('Build status')                     // 1
+  .get('/build/wercker/go-wercker-api.json') // 2
+  .expectJSONTypes(                          // 3
+    Joi.object().keys({                      // 4
+      name: 'build',                         // 5
+      value: isBuildStatus,                  // 6
+    })
+  )
 ```
 
-We need a real project to use for our tests. We'll use the programming
-language [Rust][], though we could have chosen any stable project with a
-Travis build.
-
-The `create()` method gives the tester a new test. The chained-on calls come
-from the API testing framework [IcedFrisby][]. Here's a [longer example][] and
-the complete [API guide][IcedFrisby API].
-
-`expectJSONTypes()` is an IcedFrisby method which accepts a [Joi][] schema.
+1. The `create()` method adds a new test to the tester object.
+The chained-on calls come from the API testing framework [IcedFrisby][].
+Here's a [longer example][] and the complete [API guide][IcedFrisby API].
+2. We use the `get()` method to request a badge. There are several points to consider here:
+    * We need a real project to test against. In this case we have used [wercker/go-wercker-api](https://app.wercker.com/wercker/go-wercker-api/runs) but we could have chosen any stable project.
+    * Note that when we call our badge, we are allowing it to communicate with an external service without mocking the reponse. We write tests which interact with external services, which is unusual practice in unit testing. We do this because one of the purposes of service tests is to notify us if a badge has broken due to an upstream API change. For this reason it is important for at least one test to call the live API without mocking the interaction.
+    * All badges on shields can be requested in a number of formats. As well as calling https://img.shields.io/wercker/build/wercker/go-wercker-api.svg to generate ![](https://img.shields.io/wercker/build/wercker/go-wercker-api.svg) we can also call https://img.shields.io/wercker/build/wercker/go-wercker-api.json to request the same content as JSON. When writing service tests, we request the badge in JSON format so it is easier to make assertions about the content.
```

Comment:
Currently https://img.shields.io/wercker/build/wercker/go-wercker-api.svg returns 404.

Reply:
It does, but that's because I updated this doc based on #2103 and https://github.com/badges/shields/pull/2103#issuecomment-425182915 isn't deployed yet. Once that's live, that example will render. If you want, we can block merging this until the next deploy.

Reply:
We shouldn't wait for the next deploy. This tutorial is great and can go live without this badge in my opinion.